src/test/java/stepdefinition/HerokuSD.java

```diff
@@ -6,25 +6,48 @@
 import cucumber.api.java.en.When;
 import framework.AppiumWrapper;
 import framework.page_object_model_web.HerokuPage;
+
 import java.text.ParseException;
 import java.util.concurrent.TimeUnit;
 
 public class HerokuSD {
+    private HerokuPage herokuPage = new HerokuPage();
+
+
+    @Given("^User is on the Threely login page$")
```

Comment:
there is not verification here, its an empty method

Reply:
this is till an issue since your just clicking in it, what you need to do is verify page title